calculate_baseline_metrics.py

```diff
@@ -110,12 +114,30 @@ def read_data_file(filename, channel=None, intervals=None, pad=None, dtype='floa
 
 
 def calculate_metrics(metrics, x, y):
+    """
+    Calculate metrics for given two ??TODO??.
```

Comment:
@avantikalal

Reply:
Calculate metrics for a given pair of arrays, which contain clean and noisy/denoised data.